projects/jdnconvertible-calendar-date-adapter/package.json

```diff
@@ -18,12 +18,15 @@
     "url": "https://github.com/dhlab-basel/JDNConvertibleCalendarDateAdapter/issues"
   },
   "homepage": "https://github.com/dhlab-basel/JDNConvertibleCalendarDateAdapter#readme",
+  "dependencies": {
+    "tslib": "^2.0.0"
```

Comment:
I wondered about this direct dep: https://angular.io/guide/migration-update-libraries-tslib